src/core/mac/mac_frame.hpp

```diff
@@ -1015,6 +1015,23 @@ class RxFrame : public Frame
      */
     bool IsAckedWithFramePending(void) const { return mInfo.mRxInfo.mAckedWithFramePending; }
 
+    /**
+     * This method indicates whether or not the received frame is an out-of-band frame.
```

Comment:
add doxygen to explain your design of counting Oob frame received.

Reply:
added comments

Reply:
Could you please add **REQUIREMENT CLEARLY** for platform radio driver if OobFrame is expected to be counted correctly?

Reply:
updated in the 5th commit